website/blog/2022/04/17/md.md

```diff
@@ -0,0 +1,313 @@
+---
+title: "API Observability with Apache APISIX Plugins"
+keywords:
+- Apache APISIX
+description: In this blog post, we can leverage the power of some [Apache APISIX ](https://apisix.apache.org/)Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
+tags: [Events]
+---
+
+> In this blog post, we can leverage the power of some [Apache APISIX](https://apisix.apache.org/)Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
+
+<!--truncate-->
+
+ ![Cover image for API Observability with Apache APISIX Plugins](https://res.cloudinary.com/practicaldev/image/fetch/s--Rv3WzLi8--/c_imagga_scale,f_auto,fl_progressive,h_420,q_auto,w_1000/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/9vpdkknyeky542fat56v.jpg)
+
+ [#api](https://dev.to/t/api)&nbsp;[#plugins](https://dev.to/t/plugins)&nbsp; [#monitoring](https://dev.to/t/monitoring)&nbsp; [#opensource](https://dev.to/t/opensource)
+
+In this blog post, we can leverage the power of some [Apache APISIX](https://apisix.apache.org/)Observability Plugins and take a look at how to set up these plugins, how to use them to understand API behavior, and later solve problems that impact our users.
+
+## APIs are everywhere
+
+APIs — by now, we're all familiar with the term. Every service we use today either uses an API or is an API itself. APIs are central in building and delivering your services. Also, you know that the success of your services depends on the integrity, availability, and performance of your APIs.
+
+Nowadays **API Observability** is already a part of every API development as it addresses many problems related to API consistency, reliability, and the ability to quickly iterate on new API features. When you design for full-stack observability, you get everything you need to find issues and catch breaking changes.
+
+API observability can help every team in your organization:
+
+* Sales and growth teams to monitor your API usage, free trials, observe expansion opportunities and ensure that API serves the correct data.
+
+* Engineering teams to monitor and troubleshoot API issues.
+
+* Product teams to understand API usage and business value.
+
+* Security teams to detect and protect from API threats.
+
+[![API observability in every team](https://res.cloudinary.com/practicaldev/image/fetch/s--ItJyrD63--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/h9jrwx2p2zx7nj6a4ixj.jpg)](https://res.cloudinary.com/practicaldev/image/fetch/s--ItJyrD63--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://dev-to-uploads.s3.amazonaws.com/uploads/articles/h9jrwx2p2zx7nj6a4ixj.jpg)
```

Comment:
I don't think we need every picture in the original blog, if there is no need to add or low correlation degree then ignore

Reply:
ok.